src/tablo/Cell/Cell.js

```diff
@@ -7,6 +7,8 @@ const { maybe, enums, union } = t;
 
 const propsTypes = {
   data: t.Any,
+  rowData: t.Any,
+  rowIndex: maybe(t.Integer),
```

Comment:
The problem is that `Cell` is created before without the props, then `Column` clones it and passes other props to it